doc/classes/EditorSettings.xml

```diff
@@ -387,12 +405,74 @@
 		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/joint_body_a" type="Color" setter="" getter="">
+			Color for representing [member Joint3D.node_a] for some [Joint3D] types.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/joint_body_b" type="Color" setter="" getter="">
+			Color for representing [member Joint3D.node_b] for some [Joint3D] types.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/lightmap_lines" type="Color" setter="" getter="">
+			Color of lines displayed in baked [LightmapGI] node's grid.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/lightprobe_lines" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [LightmapProbe]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/occluder" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [OccluderInstance3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/particle_attractor" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [GPUParticlesAttractor3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/particle_collision" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [GPUParticlesCollision3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/particles" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [CPUParticles3D]s and [GPUParticles3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/path_tilt" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/reflection_probe" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [ReflectionProbe]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/selected_bone" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/skeleton" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [Skeleton3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/stream_player_3d" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [AudioStreamPlayer3D]s. Used for visualizing the emission angle.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/visibility_notifier" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [VisibleOnScreenNotifier3D]s and [VisibleOnScreenEnabler3D]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_colors/voxel_gi" type="Color" setter="" getter="">
+			The 3D editor gizmo color for [VoxelGI]s.
+		</member>
+		<member name="editors/3d_gizmos/gizmo_settings/bone_axis_length" type="float" setter="" getter="">
+		</member>
+		<member name="editors/3d_gizmos/gizmo_settings/bone_shape" type="int" setter="" getter="">
+		</member>
+		<member name="editors/3d_gizmos/gizmo_settings/path3d_tilt_disk_size" type="float" setter="" getter="">
+			Size of the disk gizmo displayed when editing [Path3D]'s tilt handles.
```

Comment:
In pixels? Relative to the editor's scale?

Reply:
Seems to be roughly the radius in 3D units.